tests/integration/test_outcome.py

```diff
@@ -1277,16 +1277,6 @@ def make_envelope(transaction_name):
             "reason": "Sampled:3000",
             "source": expected_source,
         },
-        {
-            "category": 11,  # ProfileIndexed
-            "key_id": 123,
-            "org_id": 1,
-            "outcome": 1,  # Filtered
-            "project_id": 42,
-            "quantity": 1,
-            "reason": "Sampled:3000",
-            "source": expected_source,
-        },
```

Comment:
@jjbayer @Dav1dde please, can you double-check if the removal of this is correct as I'm not too familiar with this integration test.

Reply:
Yes this is correct if the profile is now forwarded to storage! (A sentry consumer would then create the accepted outcome).

Reply:
@viglia how did you manage to make this diff disappear?

Reply:
re-introduced the global kill switch, so the test made sense how it was before (if it's disabled it should be filtered out), hence I re-introduced this outcome too.